docs/Quick Start/_category_.json

```diff
@@ -0,0 +1,8 @@
+{
+  "label": "Getting Started",
+  "position": 2,
+  "link": {
+    "type": "generated-index",
+    "description": "5 minutes to learn the most important Docusaurus concepts."
```

Comment:
Change to UOR

Reply:
resolved in [7f31d4f](https://github.com/uor-framework/universalreference.io/pull/11/commits/7f31d4ffddba6c100941e3d67a841925fee8f5d1)